recipes/spirv-tools/all/conanfile.py

```diff
@@ -138,45 +135,88 @@ def package(self):
             tools.remove_files_by_mask(os.path.join(self.package_folder, "bin"), "*SPIRV-Tools-shared.dll")
             tools.remove_files_by_mask(os.path.join(self.package_folder, "lib"), "*SPIRV-Tools-shared*")
 
+        if self.options.shared:
+            targets = {"SPIRV-Tools-shared": "spirv-tools::SPIRV-Tools"}
+        else:
+            targets = {
+                "SPIRV-Tools": "spirv-tools::SPIRV-Tools", # before 2020.5, kept for conveniency
+                "SPIRV-Tools-static": "spirv-tools::SPIRV-Tools",
+                "SPIRV-Tools-opt": "spirv-tools::SPIRV-Tools-opt",
+                "SPIRV-Tools-link": "spirv-tools::SPIRV-Tools-link",
+                "SPIRV-Tools-reduce": "spirv-tools::SPIRV-Tools-reduce",
+            }
+        self._create_cmake_module_alias_targets(
+            os.path.join(self.package_folder, self._module_file_rel_path),
+            targets,
+        )
+
+    @staticmethod
+    def _create_cmake_module_alias_targets(module_file, targets):
+        content = ""
+        for alias, aliased in targets.items():
+            content += textwrap.dedent("""\
+                if(TARGET {aliased} AND NOT TARGET {alias})
+                    add_library({alias} INTERFACE IMPORTED)
+                    set_property(TARGET {alias} PROPERTY INTERFACE_LINK_LIBRARIES {aliased})
+                endif()
+            """.format(alias=alias, aliased=aliased))
+        tools.save(module_file, content)
+
+    @property
+    def _module_subfolder(self):
+        return os.path.join("lib", "cmake")
+
+    @property
+    def _module_file_rel_path(self):
+        return os.path.join(self._module_subfolder,
+                            "conan-official-{}-targets.cmake".format(self.name))
+
     def package_info(self):
+        self.cpp_info.filenames["cmake_find_package"] = "SPIRV-Tools"
+        self.cpp_info.filenames["cmake_find_package_multi"] = "SPIRV-Tools"
         self.cpp_info.names["pkg_config"] = "SPIRV-Tools-shared" if self.options.shared else "SPIRV-Tools"
-        # FIXME: official CMake imported targets are not namespaced
```

Comment:
Perhaps keep a `FIXME` with the missing targets/config files about which you've written in your initial comment?

Reply:
I've kept another FIXME about missing config files (imported targets are all good now).